notification/src/test/java/io/skai/notification/service/impl/NotificationServiceImplTest.java

```diff
@@ -0,0 +1,64 @@
+package io.skai.notification.service.impl;
+
+import io.skai.notification.config.GmailProperties;
+import io.skai.notification.model.Notification;
+import io.skai.notification.model.Template;
+import io.skai.notification.service.EmailService;
+import io.skai.notification.service.NotificationService;
+import io.skai.notification.service.TemplateService;
+import org.junit.jupiter.api.Test;
+
+import java.util.IllegalFormatException;
+
+import static io.skai.notification.enums.OrderStatus.NEW;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.*;
+
+class NotificationServiceImplTest {
+    TemplateService templateService = mock(TemplateService.class);
+    EmailService emailService = mock(EmailService.class);
+    GmailProperties gmailProperties = mock(GmailProperties.class);
+    NotificationService notificationService = new NotificationServiceImpl(templateService, emailService, gmailProperties);
+
+    @Test
+    void shouldReturnBody() {
+        String template = "--%1$s--%2$s--%3$s--%4$s--%5$s--%6$s--%7$s";
+        String expected = "--one--two--three--four--five--six--seven";
+        String result = String.format(template, "one", "two", "three", "four", "five", "six", "seven");
+        assertThat(result)
```

Comment:
you should check your business logic instead of how String.format() works

Reply:
my fail. i will change it